### ISSUE TITLE
Probleme with --force-hardsub argument

### ISSUE THREAD

**Issue by nirre55** [crunchy-labs/crunchy-cli#262]
**Describe the bug**
when I try to force subtitles I get this error message :
            - Error applying option 'original_size' to filter 'ass': Invalid argument
            Invalid argument
            
but when I remove the argument --force-hardsub, its work, if this is a known bug, is there a way to recover the subtitle files (.SRT) linked to the video directly? 

**To Reproduce**
command to reproduce the behavior:
```
$ crunchy download --force-hardsub -a ja-JP -s en-US https://www.crunchyroll.com/watch/G7PU4MZ04/red-scale
```

**Comment by ByteDream** [comment]
I sadly can't reproduce this issue.
You can extract the subtitles by running `ffmpeg -i <downloaded file> subtitles.srt`

**Comment by ByteDream** [comment]
Fixed in [v3.1.0](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v3.1.0).
